internal/constants/constants.go

```diff
@@ -160,10 +160,6 @@ const ArtifactCacheFileName = "artifact_cache.json"
 // ArtifactMetaDir is the directory in which we store meta information about artifacts
 const ArtifactMetaDir = "artifacts"
 
-// ArtifactCacheSizeEnvVarName is the maximum size in MB of the artifact cache.
-// The default is 500MB.
-const ArtifactCacheSizeEnvVarName = "ACTIVESTATE_ARTIFACT_CACHE_SIZE_MB"
```

Comment:
Seeing as we deferred handling this via config I think it makes sense to keep this in.